projects/uitdatabank/toc.json

```diff
@@ -50,7 +50,15 @@
     {
       "type": "item",
       "title": "Entry API",
-      "uri": "reference/entry.json"
+      "uri": "reference/entry.json",
+      "item": [
+        {
+          "type": "item",
+          "title": "Project flow from test to production",
```

Comment:
Maybe something like `Requirements before going live` is clearer at first glance?

Reply:
I agree